Equipment.md

```diff
@@ -124,26 +126,30 @@ __Vulnerability__ is an optional rule for players who want excessive realism. It
 
 |Armor|AC Bonus|Move Cost|Vulnerability|
 |---|---|---|---|
-|  Light Leather   |+2 |0 ft. |
+|  Light Leather   |+2 |0 ft. |5 [Slashing]
 | Light Piecemeal  |+3 |0 ft. |
 |   Light Metal    |+3 |5 ft. |
 |   Light Hybrid   |+4 |0 ft. |
-|  Medium Leather  |+5 |0 ft. |
+|  Medium Leather  |+5 |0 ft. |2 [Slashing]
 | Medium Piecemeal |+6 |0 ft. |2 [Piercing]
-|   Medium Metal   |+7 |5 ft. |
+|   Medium Metal   |+7 |5 ft. |2 [Bludgeoning]
 |  Medium Hybrid   |+8 |5 ft. |
 |  Heavy Leather   |+9 |0 ft. |
 | Heavy Piecemeal  |+10|5 ft. |5 [Piercing]
 |   Heavy Metal    |+11|10 ft.|5 [Bludgeoning]
 |   Heavy Hybrid   |+12|10 ft.|
+
+|Shield|AC Bonus|Move Cost|Vulnerability|
+|---|---|---|---|
 |Light Wood Shield |+1 |0 ft. |
-|Light Metal Shield|+1 |0 ft. |
+|Light Metal Shield|+2 |0 ft. |
```

Comment:
I had these identical because material does not hugely affect its ability to stop a blow (because metal shields were made thinner to account for their increased weight). The main concern of wood/metal in my mind was "what happens when a dragon breathes fire on you? How about lightning?". Didn't want to canonize that in rules, but wanted to give that option to GMs.

I think we should either use the Vulnerability rules here or have difference AC values, but not both. Slightly leaning toward AC values (at least for now).

Reply:
Sounds good.  Changing it back to 1 and 3 for both types, I'll add vulnerabilities for now.